bvm/ballerina-core/src/main/resources/MessagesBundle.properties

```diff
@@ -151,6 +151,9 @@ task.not.running = The task in not running
 illegal.format.conversion = illegal format conversion ''{0}''
 incompatible.stamp.operation = incompatible stamp operation: ''{0}'' value cannot be stamped as ''{1}''
 cannot.stamp.null = cannot stamp ''null'' value to type ''{0}''
+cannot.convert.null = cannot convert ''null'' value to type ''{0}''
```

Comment:
nil instead of null ?

Reply:
Added same error message similar to existing error message in stamp. I think we need to change both. So I will create issue and will send a pr to fix that separately.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/14076